### PR TITLE
Feat keyboard focus

### PR DESCRIPTION
**Pull request by azuldev1**
Had trouble creating a lack of keyboard focus within Stele. Making it difficult to test this behavior. I added:

// Ensure the application window has focus as well as the embedded content // will be called on settings page and when url is switched to home url mainWindow.webContents.on('dom-ready', () => { console.log('DOM READY'); mainWindow.focus(); mainWindow.webContents.focus(); });

The idea is that maybe within keyboard/keyboard encoder dependent interactives either the Wrapper Application or the embedded web app does not have focus. With something like a touch screen based interactive the initial touch of the screen could be calling the two different layers of the application into focus. I added the event listener above, which should call both layers into focus.

Used the IAT App to test, the screen saver for this app is listening for .on keydown event, which means only keyboard interaction will get you passed the screen saver view. Did not have any issues getting past the screen saver on windows or mac. Tested via application launching at startup and ensuring no initial mouse interactions.

### CODE REVIEW

File: src/main.dev.js [hunk 1]
```diff
@@ -248,6 +248,14 @@ app.on('ready', async () => {
     }
   });
 
+  // Ensure the application window has focus as well as the emmbedded content
```
Comment (bryankennedy) [inline src/main.dev.js]: Fix spelling `emmbedded`

File: src/main.dev.js [hunk 2]
```diff
@@ -248,6 +248,14 @@ app.on('ready', async () => {
...
+  // will be called on settings page and when url is switched to home url
+  mainWindow.webContents.on('dom-ready', () => {
+    console.log('DOM READY');
```
Comment (bryankennedy) [inline src/main.dev.js]: Remove console.log

I assume that you're doing this here for dev purposes. If you think it's worth logging this state to the log file then make sure to use logger.info or some other state, and match the style of other log messages in the same file.

File: src/main.dev.js [hunk 3]
```diff
@@ -248,6 +248,14 @@ app.on('ready', async () => {
...
+    mainWindow.focus();
```
Comment (bryankennedy) [inline src/main.dev.js]: We should add a wrapper around this to only fire on production. See some previous commits for history.
https://github.com/scimusmn/stele/commit/8b81666d74fb84a4fafa02a90a1c891d05172cf1

Essentially we don't want to re-introduce this bug: https://github.com/scimusmn/stele/issues/23

File: src/main.dev.js [hunk 4]
```diff
@@ -248,12 +248,13 @@ app.on('ready', async () => {
...
   });
 
-  // Ensure the application window has focus as well as the emmbedded content
+  // Ensure the application window has focus as well as the embedded content
   // will be called on settings page and when url is switched to home url
   mainWindow.webContents.on('dom-ready', () => {
-    console.log('DOM READY');
-    mainWindow.focus();
-    mainWindow.webContents.focus();
+    if (process.env.NODE_ENV === 'production'){
```
Comment (azuldev1) [inline src/main.dev.js]: added production check for .focus logic

**Comment by azuldev1** [comment]
I merged in master because it was a few commits ahead of this branch. Merging should go smoothly once approved.

**Pull request merged**